### PR TITLE
feat:[TMSF-14] - add authentication support with JWT

### PR DESCRIPTION
**Pull request by as-ajitsingh**

### CODE REVIEW

**Comment by as-ajitsingh** [comment]
Will fix pnpm issue when finalizing the PR.

File: src/auth/strategy/jwt-strategy.ts
Comment (as-ajitsingh) [inline src/auth/strategy/jwt-strategy.ts:1]: Export through Barrel Export.

File: yarn.lock
Comment (as-ajitsingh) [inline yarn.lock:1]: Remove yarn.lock and update pnpm

**Review by ravindermahajan890 — APPROVED** [review]

**Comment by as-ajitsingh** [comment]
@GokulnathP Please work on these things: 

- fix e2e test failure in pipeline
- env configuration
- Prisma generate as a part build process
- Make Application work in Prod

**Pull request merged**